fighters/robin/src/opff.rs

```diff
@@ -32,6 +32,26 @@ unsafe fn elwind1_burn(fighter: &mut L2CFighterCommon) {
     }
 }
 
+unsafe fn levin_leniency(fighter: &mut L2CFighterCommon, boma: &mut BattleObjectModuleAccessor) {
+    let levin = *FIGHTER_REFLET_INSTANCE_WORK_ID_INT_THUNDER_SWORD_CURRENT_POINT;
```

Comment:
This line should be placed inside the if block so it doesn't get run every frame (move the check for the point total inside it as well)